__tests__/app.test.js

```diff
@@ -164,3 +164,40 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("POST: 201 - responds with a 201 status with a newly added comment to the db", () => {
+    const newComment = {
+      username: "dav3rid",
+      body: "first comment woop",
+    };
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment.body).toBe("first comment woop");
+      });
+  });
```

Comment:
Just need another happy path test here to confirm that extra keys on the newComment body are ignored if the rest of the request is good.